.gitignore

```diff
@@ -1,4 +1,4 @@
-config.toml
+# VSCode
```

Comment:
It feels a bit weird having user specific stuff in the project's `.gitignore` (users can have their own config in their `~/.gitignore`).

I know you only added the comment, not the line itself, so i'm not asking you to remove it, just a side note ;)

Reply:
You mean stuff related to a specific IDE in a `.gitignore` file? You think `.gitignore` should be IDE-agnostic? I think it can host stuff related to several possible dev environments, but open to discuss!

Reply:
yeah, it's kinda weird having IDE specific stuff in the project's `.gitignore` file, because we might then want to add all sorts of other IDE quirks and stuff... I'm not against it, I just find it odd